jobilla/appevents/options.proto

```diff
@@ -1,5 +1,8 @@
 syntax="proto3";
+
 option go_package="github.com/jobilla/protoc-gen-go-appevents/jobilla/appevents";
+option php_metadata_namespace = "GPBMetadata\\Jobilla\\Appevents\\Options";
```

Comment:
you need to add the `php_namespace` too. You should also commit the generated PHP files.